docs/editors/vim.md

```diff
@@ -44,13 +43,9 @@ curl --compressed -o- -L https://yarnpkg.com/install.sh | bash
 
 ## Installing coc.nvim
 
-Once the requirements are satisfied, we can now proceed to install the following
-plugins:
-
-- [`neoclide/coc.nvim`](https://github.com/neoclide/coc.nvim/): Language Server
-  Protocol client to communicate with the Metals language server.
-- [`derekwyatt/vim-scala`](https://github.com/derekwyatt/vim-scala): for syntax
```

Comment:
I removed this because of 2 reason.

1. The most useful part of this is the syntax highlighting. All modern versions of vim/nvim ship with the syntax highlight from this plugin by default.
2. The only other thing you'd probably use this for is to sort imports, but it's all done via regex. I'd rather have someone use scalafix.